Styles polishing (#17)
actum/aria-hooks: src/hooks/tooltip/tooltip.stories.mdx

<Meta title="Aria Component/Tooltip" />

# Tooltip

> Component that displays additional information about parent element. It is visible only when user hovering over parent element, or when parent element is focused.
> 
> Most commonly displays description of a button's function or hint on keyboard shortcut to activate functionality.

Reviewer: ```suggestion
A component that displays informative text on hover, focus, or tap.

It can be used to display a description of a button's function or hint on keyboard shortcut to activate functionality.
```
